src/components/policy/policy_regular/src/sql_pt_representation.cc

```diff
@@ -1177,9 +1177,11 @@ bool SQLPTRepresentation::SaveConsumerFriendlyMessages(
   // the policy table. So it won't be changed/updated
   if (messages.messages.is_initialized()) {
     utils::dbms::SQLQuery query(db());
-    if (!query.Exec(sql_pt::kDeleteMessageString)) {
-      LOG4CXX_WARN(logger_, "Incorrect delete from message.");
-      return false;
+    if (!messages.messages->empty()) {
```

Comment:
@AKalinich-Luxoft 
I think it would be better to avoid deep nesting e.g. like
```

if (!msg.is_initialized()) {
   return false;
}
if (msg.empty()) {
   return false;
}
<process messages>

```